jupyter_server/event_schemas/kernels/v1.yaml

```diff
@@ -0,0 +1,50 @@
+"$id": https://events.jupyter.org/jupyter_server/kernels/v1
+version: 1
+title: Kernel Manager activities
+personal-data: true
+description: |
+  Record events of a kernel manager.
+type: object
+required:
+  - action
+  - kernel
+  - message
+properties:
+  action:
+    enum:
+      - start
+      - interrupt
+      - shutdown
+      - restart
+      - error
+    description: |
+      Action performed by the Kernel Manager.
+
+      This is a required field.
+
+      Possible values:
+
+      1. start
+         A kernel has been started with the given kernel id.
+
+      2. interrupt
+         A kernel has been interrupted for the given kernel id.
+
+      3. shutdown
+         A kernel has been shut down for the given kernel id.
+
+      4. restart
+         A kernel has been restarted for the given kernel id.
+
+      5. error
+         A kernel has been errored out.
+  kernel:
+    type: string
+    description: |
+      Kernel id.
+
+      This is a required field.
+  message:
+    type: string
+    description: |
+      Description of the event specified in action.
```

Comment:
Would it make sense to introduce a `kernel_server` or `server` property.  For Gateway-based kernels, this would be the host value parsed from the `gateway-url` and `localhost` (or the host parsed out of the initial url) for all others.

Reply:
Adding server property make sense to me. Included the details.